docker/pudl_etl.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+
+pudl_setup \
+    --pudl_in $CONTAINER_PUDL_IN \
+    --pudl_out $CONTAINER_PUDL_OUT
+ferc1_to_sqlite \
+    --clobber \
+    --gcs-cache-path gs://zenodo-cache.catalyst.coop \
+    --bypass-local-cache \
+    --logfile $LOGFILE \
+    $PUDL_SETTINGS_YML
+pudl_etl \
+    --clobber \
+    --gcs-cache-path gs://zenodo-cache.catalyst.coop \
+    --bypass-local-cache \
+    --logfile $LOGFILE \
+    $PUDL_SETTINGS_YML
+
+# Shut down the deploy-pudl-vm instance when the etl is done.
+ACCESS_TOKEN=`curl \
+    "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token" \
+    -H "Metadata-Flavor: Google" | jq -r '.access_token'`
+
```

Comment:
Is there any reason why we can't do something like this to publish the data?
```
gsutil -m cp -r $CONTAINER_PUDL_OUT gs://intake.catalyst.coop/$GHREF
```
I'm not sure how ask git for the refs that point to the current commit though.